analyses/independent-samples/independent-samples.R

```diff
@@ -19,14 +19,14 @@
 #' 
 #' @return a data frame of Participant and Specimen IDs, each present only once.
 independent_samples <- function(sample_df, 
-                                tumor_types = c("primary", "prefer_primary", "any"), 
+                                tumor_types = c("primary", "secondary", "prefer_primary", "any"), 
                                 seed){
   tumor_types <- match.arg(tumor_types)
   if(!missing(seed)){set.seed(seed)}
   
-  primary_descs <- c("Initial CNS Tumor", "Diagnosis")
+  primary_descs <- c("Initial CNS Tumor", "Diagnosis", "Primary tumor")
```

Comment:
```suggestion
  primary_descs <- c("Initial CNS Tumor", "Primary tumor")
```
we have removed the Diagnosis nomenclature from our database, so we can remove it here